mono/utils/mono-merp.c

```diff
@@ -97,7 +97,7 @@ typedef struct {
 	const char *osVersion; 
 	int uiLidArg; // Application LCID 
 
-	const char systemModel [100];
+	char systemModel [100];
```

Comment:
Real problem.

Reply:
What's the problem here? all uses of systemModel are either okay with `const` or cast constness away.

I'm not opposed to this change, but I'm curious what it's fixing.

Reply:
Error message is in the commit. Having const members requires initializing them, once — or multiple times and casting away const as you say. There may be another fix.

Reply:
I’m really torn on this. More const seems good, even if interface-only. Lack of these errors in general suggests it is rare. And not idiomatic. But still good. Rare != bad, often the opposite. New/better implies rare.

The secondary reason is a recent msvc compiler regression and will not stand long, did ship, does not hit us, probably never will.